docs/reference/commandline/node_drain.md

```diff
@@ -0,0 +1,27 @@
+<!--[metadata]>
++++
+title = "node drain"
+description = "The node drain command description and usage"
+keywords = ["node, drain"]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+# node drain
+
+    Usage:  docker node drain NODE [NODE...]
+
+    Drain a node in the swarm
+
+Drain a node into the swarm. This command targets a docker engine that is a manager in the swarm cluster.
```

Comment:
Explain what draining means: all tasks will be removed from this node, and no tasks will be assigned to it until it is reactivated with "docker node activate".

Same comment as above about "This command targets a docker engine...".